client/src/pages/Auth/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useNavigate } from "react-router-dom";
+import SingupContainer from "./Signup";
+import SignupRedirect from "./Login/SignupRedirect";
+import LoginRedirect from "./Signup/LoginRedirect";
+import LoginContainer from "./Login";
+import { useState } from "react";
+
+function Auth() {
+    const navigate = useNavigate();
+    const [isLoggingIn, setIsLoggingIn] = useState(true)
+
+    function handleLogin(): void {
+        navigate('/dashboard');
```

Comment:
if we have a unique file for the routes path, we can just import the paths and use them here instead